src/Layout.tsx

```diff
@@ -45,7 +45,7 @@ const Layout = ({
 
 	return <TAG className={containerClass}>
 		{!childrenAfterRegions ? children : null}
-		{regionsData ? <Regions regionsData={regionsData} classes={regionClasses} names={regionNames}/> : null}
+		{regionsData && <Regions regionsData={regionsData} classes={regionClasses} names={regionNames}/>}
```

Comment:
If regionsData is for example the number 0, it will be printed...

https://developer.mozilla.org/en-US/docs/Glossary/Falsy